tests/data/config/atcamera_dewar/README.txt

```diff
@@ -0,0 +1,6 @@
+Configuration files for the ATCameraDewar rule unit tests.
```

Comment:
I like what you did here. We should make this common practice to have an "index" of the different test configs.

Reply:
I do this for most rules (especially the rules with complicated configuration).